pkg/cmd/step/helm/step_helm_apply.go

```diff
@@ -389,6 +389,11 @@ func (o *StepHelmApplyOptions) Run() error {
 func (o *StepHelmApplyOptions) getRequirements() (*config.RequirementsConfig, string, error) {
 	// Try to load first the requirements from current directory
 	requirements, requirementsFileName, err := config.LoadRequirementsConfig(o.Dir)
+
+	_, err = os.Stat(requirementsFileName)
```

Comment:
I think that we should skip the `Stat` when the `err` returned by the `LoadRequirementsConfig` is not nil, and directly load the requirements from the team settings.

Reply:
The thing is that LoadRequirementsConfig IMHO should return an error when `jx-requirments.xml` was not found. The current implementation will never return an error because it's swallowed by [FileExists function](https://github.com/jenkins-x/jx/blob/6f606365f4ce912207b1b822f8426e7e037e9f6c/pkg/util/files.go#L43-L45). Then the [LoadRequirementsConfigFile](https://github.com/jenkins-x/jx/blob/6f606365f4ce912207b1b822f8426e7e037e9f6c/pkg/config/install_requirements.go#L583-L586) will return an error which is nil. Could you check my explanation in the issue #6653? 

I'm not sure if changing files.FileExists to not swallow the error is the correct approach to solve this issue.

Reply:
I agree that it should return an error when the file doesn't exit. Not quite sure if there are any places in boot relying on this default requirements when file doesn't exit. 

It is worth trying to change the `LoadRequirementsConfigFile` to return an error when the file doesn't exit.

Reply:
I agree. Today I will try to fix that.